src/Api/Vault/AuthorizationHandlers/Collections/CollectionAuthorizationHandler.cs

```diff
@@ -80,7 +80,7 @@ public CollectionAuthorizationHandler(ICurrentContext currentContext, ICollectio
         // Owners, Admins, Providers, and users with CreateNewCollections or EditAnyCollection permission can always create collections
```

Comment:
Comment needs updating 😁 
In fact, I suggest you can just delete it because the `if` statement is pretty clear